code/modules/mob/living/simple_animal/slime/slime_type.dm

```diff
@@ -0,0 +1,198 @@
+/datum/slime_type
+	///Our slime's colour as text. Used by both description, and icon
+	var/colour
+	///The type our slime spawns
+	var/core_type
+	///The possible mutations of our slime
+	var/list/mutations
+
+	var/static/list/slime_colours_to_rgb = list(
```

Comment:
Should probably doc this

Reply:
On further note, this slime_colour_to_rgb is superflous. I can just add it as a var to the slime type, which is less complicated, and it will make the slime colour not depend on the string define.